sdk/identity/Azure.Identity/src/Credentials/BrowserCustomizationOptions.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+using Microsoft.Identity.Client;
+
+namespace Azure.Identity
+{
+    /// <summary>
+    /// Options to customize browser view.
+    /// </summary>
+    public class BrowserCustomizationOptions
+    {
+        /// <summary>
+        /// Specifies if the public client application should used an embedded web browser
+        /// or the system default browser
+        /// </summary>
+        public bool? UseEmbeddedWebView;
+
+        internal SystemWebViewOptions SystemBrowserOptions;
+
+        internal EmbeddedWebViewOptions EmbeddedBrowserOptions;
+
+        private SystemWebViewOptions systemWebViewOptions
+        {
+            get
+            {
+                SystemBrowserOptions ??= new SystemWebViewOptions();
+                return SystemBrowserOptions;
+            }
+        }
+
+        private EmbeddedWebViewOptions embeddedWebViewOptions
+        {
+            get
+            {
+                EmbeddedBrowserOptions ??= new EmbeddedWebViewOptions();
+                return EmbeddedBrowserOptions;
+            }
+        }
+
+        /// <summary>
+        /// Property to set HtmlMessageSuccess of SystemWebViewOptions.
```

Comment:
This comment isn't helpful. I'd suggest explaining its purpose without using code member names.